src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationapi/service/ClientDataService.kt

```diff
@@ -25,6 +27,15 @@ class ClientDataService(
     return allClients.map { client -> mapToClientDetails(client, allClientConfigsMap) }
   }
 
+  fun getAllClientsWithLastAccessed(): List<ClientLastAccessedResponse> =
+    clientRepository.findAll().map {
+      ClientLastAccessedResponse(
+        it.clientId,
+        it.getLastAccessedDate().atZone(ZoneId.systemDefault()).toLocalDateTime(),
+        System.getenv("HOSTNAME") ?: "unknown",
```

Comment:
Better to capture host name outside of this loop? Maybe when the function is called, or even when the bean is initialised?